packages/search-types/src/suggestion.model.ts

```diff
@@ -9,13 +9,14 @@ import { Previewable } from './previewable.model';
  * @public
  */
 export interface Suggestion extends NamedModel, Previewable {
-  /** {@inheritDoc Previewable.facets} */
+  // eslint-disable-next-line tsdoc/syntax
+  /** {@inheritDoc Previewable.facets}. */
```

Comment:
Same case as before.

```suggestion
  // eslint-disable-next-line jsdoc/require-description-complete-sentence
  /** {@inheritDoc Previewable.facets} */
```